x/zoneconcierge/keeper/ibc_packet_btc_headers.go

```diff
@@ -18,35 +19,47 @@ func (k Keeper) BroadcastBTCHeaders(ctx context.Context) error {
 		return nil
 	}
 
-	// Current behavior:
-	// - If no headers sent: Returns last w+1 headers from tip
-	// - If headers previously sent:
-	//   - If last segment valid: Returns headers from last sent to tip
-	//   - If last segment invalid (reorg): Returns last w+1 headers from tip
-	//
-	// TODO: Should use Consumer base BTC header as starting point:
-	// - If no headers sent: Return from Consumer base to tip
-	// - If headers previously sent:
-	//   - If last segment valid: Return from last sent header to tip
-	//   - If last segment invalid (reorg): Return from Consumer base to tip
-	headers := k.getHeadersToBroadcast(ctx)
-	if len(headers) == 0 {
-		k.Logger(sdkCtx).Info("skipping BTC header broadcast",
-			"reason", "no headers to broadcast",
-		)
-		return nil
-	}
+	// New behavior using Consumer-specific base headers:
+	// - If no Consumer base header exists: fallback to sending k+1 from tip
+	// - If Consumer base header exists but no headers sent: from Consumer base to tip
+	// - If headers previously sent: from child of most recent valid header to tip
+	// - If reorg detected: from Consumer base to tip
```

Comment:
I feel like it's not efficient to handle reorg here. We probably need to send a reorg event and send from the reorged header to tip? Maybe a TODO for now